test/command/suite/normalizers/nfkc100/unify_kana_prolonged_sound_mark/hiragana/a.expected

```diff
@@ -0,0 +1,117 @@
+normalize   'NormalizerNFKC100("unify_kana_prolonged_sound_mark", true,                      "report_source_offset", true)'   "あーぁーいーぃーうーぅーえーぇーおーぉー"   WITH_CHECKS|WITH_TYPES
+[
+  [
+    0,
+    0.0,
+    0.0
+  ],
+  {
+    "normalized": "ああぁあいいぃいううぅうええぇえおおぉお",
```

Comment:
Should we use `あ` instead of `ぁ` for `ぁー`?

Reply:
Do you mean `ぁー` should be normalized to `ぁぁ`, or `ああ` or something else?

Reply:
`ぁぁ`. Why do you choose `ぁあ`? Easy to implement?

Reply:
> Why do you choose `ぁあ?` Easy to implement?

Because `ぁぁ` is not pronounceable in Japanese.
Some one uses `ぁぁ` like `ぎゃぁぁぁ`, but I think it is broken Japanese.

Reply:
For example, the pronunciation of `ファー` is closer to `ファア` than `ファァ`.

Reply:
I understand. I'm OK with `ああ` then.

Reply:
OK, I will fix them.

Reply:
Ah, sorry. `ぁあ`.

Reply:
Oh, I see.